### PR TITLE
Remove wallet.fiatCurrencyCode usage

### PR DESCRIPTION
**Pull request by Jon-edge**
Also removes the fiat currency selection from wallet create/import/split UI

### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207279620798011

### CODE REVIEW

File: src/components/themed/TransactionListTop.tsx [hunk 1]
```diff
@@ -792,13 +792,14 @@ export function TransactionListTop(props: OwnProps) {
   const dispatch = useDispatch()
   const account = useSelector(state => state.core.account)
   const exchangeRates = useSelector(state => state.exchangeRates)
+  const { defaultFiat, defaultIsoFiat } = useSelector(state => state.ui.settings)
```
Comment (swansontec) [inline src/components/themed/TransactionListTop.tsx:795]: Same.

File: src/components/themed/WalletListSortableRow.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ function WalletListSortableRowComponent(props: Props) {
   const dispatch = useDispatch()
 
   const showBalance = useSelector(state => state.ui.settings.isAccountBalanceVisible)
-  const defaultIsoFiat = useSelector(state => state.ui.settings.defaultIsoFiat)
+  const { defaultFiat, defaultIsoFiat } = useSelector(state => state.ui.settings)
```
Comment (swansontec) [inline src/components/themed/WalletListSortableRow.tsx:33]: Drop `defaultFiat` and just pass `defaultIsoFiat` to `getSymbolFromCurrency`.

**Review by swansontec — CHANGES_REQUESTED** [review]

File: src/components/scenes/TransactionDetailsScene.tsx [hunk 3]
```diff
@@ -83,30 +83,27 @@ const TransactionDetailsComponent = (props: Props) => {
   // #region Crypto Fiat Rows
 
   // Look up wallet stuff:
-  const isoFiatCurrencyCode = useWatch(wallet, 'fiatCurrencyCode')
+  const { defaultFiat, defaultIsoFiat } = useSelector(state => state.ui.settings)
   const walletName = useWatch(wallet, 'name')
-  const fiatCurrencyCode = isoFiatCurrencyCode.replace('iso:', '')
-  const fiatSymbol = getSymbolFromCurrency(fiatCurrencyCode)
+  const fiatSymbol = getSymbolFromCurrency(defaultFiat)
```
[the remark this answers is not in the document]
Comment (Jon-edge) [reply src/components/scenes/TransactionDetailsScene.tsx:88]: Good catch, guess we had a bug

File: src/components/themed/FioRequestRow.tsx [hunk 4]
```diff
@@ -197,7 +198,9 @@ const emptyDenomination = { name: '', multiplier: '0' }
 export const FioRequestRow = connect<StateProps, {}, OwnProps>(
   (state, ownProps) => {
     const { fioRequest } = ownProps
-    const { defaultFiat, defaultIsoFiat } = state.ui.settings
+    const defaultIsoFiat = useSelector(state => state.ui.settings.defaultIsoFiat)
+    const defaultFiat = removeIsoPrefix(defaultIsoFiat)
```
Comment (swansontec) [inline src/components/themed/FioRequestRow.tsx:202]: Get rid of this `defaultFiat`, and just pass `defaultIsoFiat` to `getFiatSymbol`

**Review by swansontec — APPROVED** [review]
One change got skipped, but it's so small I think we can approve this.

File: src/components/themed/FioRequestRow.tsx (hunk 4 above)
Comment (Jon-edge) [reply src/components/themed/FioRequestRow.tsx:202]: Got it

**Pull request merged**